demo/kubernetes/rook-pool.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rook.io/v1beta1
+kind: Rookpool
+metadata:
+  name: rook-replicapool
+spec:
+  name: replicapool
+  namespace: rook
+  replication:
+    count: 3
```

Comment:
this replica count and the erasure coding values below should probably be less so they work on the vagrant single node env.  If the replication count is greater than the number of OSDs then you'll get degraded/stuck placement groups.